src/Umbraco.Community.Contentment/DataEditors/DataList/DataSources/XmlDataListSource.cs

```diff
@@ -81,8 +92,8 @@ public IEnumerable<DataListItem> GetItems()
 
             foreach (XmlNode node in nodes)
             {
-                var name = node.SelectSingleNode(nameXPath);
-                var value = node.SelectSingleNode(valueXPath);
+                var name = node.SelectSingleNode(nameXPath, ns);
+                var value = node.SelectSingleNode(valueXPath, ns);
```

Comment:
The `SelectSingleNode` calls for the icon and description (lower down) are missing the namespace-manager `ns`.